jp2_pc/Source/Lib/GeomDBase/WaveletQuadTree.cpp

```diff
@@ -523,7 +523,7 @@ namespace NMultiResolution
 
 
 	// Constant used in expressions to avoid crummy code generation by the fucked-up compiler.
-	static const CQuadNodeTIN::TState stMASK_TRI = Set(CQuadNodeTIN::estLEAF) + CQuadNodeTIN::estLEAF_COMBINE;
+	static const CQuadNodeTIN::TState stMASK_TRI = CQuadNodeTIN::TState() + CQuadNodeTIN::estLEAF + CQuadNodeTIN::estLEAF_COMBINE;
 
```

Comment:
For some unclear reason, the `Set()` factory function from `Lib/Std/Set.hpp` does not work here. That function is substituted by "inlining" its content.